marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/flow/testTargetCollectionAdditivity/flowWithTargetCollectionsAdditivity.sjs

```diff
@@ -0,0 +1,31 @@
+const test = require("/test/test-helper.xqy");
+const hubTest = require("/test/data-hub-test-helper.sjs");
+const flowRunner = require("/data-hub/5/flow/flowRunner.sjs");
+
+let assertions = [];
+let results, collections;
+
+const options = {
+  sourceDatabase: "data-hub-FINAL",
+  targetDatabase: "data-hub-FINAL",
+  targetCollectionsAdditivity: true,
+  outputFormat: 'json'
+};
+
+const content = [{uri:"/customer1.json",
+  value:{"hello": "world"},
+  context: {originalCollections: ["source"]}
+}];
+results = flowRunner.processContentWithFlow("simpleMappingFlow", content, "1", options);
+collections = hubTest.getRecord("/customer1.json").collections;
+
+assertions.push(
+  // doc should have been written
```

Comment:
With tests, assertion messages are more valuable than comments, as they explain why something is expected and they'll show up in the failure. So instead of a comment, just do:

    test.assertEqual("finished", results.jobStatus)

And no message is really needed there, it's clear what is expected.